Update documentation to reflect feature flag state (#13694)
PrairieLearn/PrairieLearn: docs/courseInstance/index.md

3. They can be invited to a course instance by an instructor. Instructors can invite students to a course instance by visiting the "Students" tab of the course instance and clicking the "Invite" button. Invites will show up on the student's PrairieLearn homepage.

> !!! tip "Course listing page has been removed"

Reviewer: Note to ourselves: we should remove this in a month or two once we've been through a cycle of people starting their courses for a new term.

@reteps consider a tracking issue for this?

Author: #13705

>     The courses listing page was removed in January 2026 in favor of the new self-enrollment system. Instructors should transition to providing a direct link to their course instance or a self-enrollment code to students. You can find this link in your course instance settings.

Reviewer: Wording nits:

```suggestion
    The courses listing page was removed in January 2026 in favor of the new self-enrollment system. Instructors should provide their students with either an enrollment code or a direct link to their course instance. Both of these can be found on the course instance settings page.
```
